test/LockTest.js

```diff
@@ -256,7 +262,7 @@ contract('Lock', ([rootAccount, ...accounts]) => {
 
   describe('app not initialized', () => {
     it('reverts on forwarding', async () => {
-      await assertRevert(lockForwarder.forward('0x', { from: rootAccount }), 'LOCK_CAN_NOT_FORWARD')
+      await assertRevert(lockForwarder.forward('0x', { from: rootAccount }))
```

Comment:
You can put the error message back once you make the changes I've suggested. Alternatively I believe you could have put the `APP_AUTH_FAILED` message here.